Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md

```diff
@@ -23,22 +23,26 @@ for more information.
 
 ## Static Provisioning
 
-Both drivers also support the creation of static PV and static PVC from existing RBD image/CephFS volume. Refer to [static PVC](https://github.com/ceph/ceph-csi/blob/devel/docs/static-pvc.md) for more information.
+The RBD and CephFS rivers support the creation of static PVs and static PVCs from
+an existing RBD image or CephFS volume. Refer
+o [static PVC](https://github.com/ceph/ceph-csi/blob/devel/docs/static-pvc.md) for more information.
 
 ## Configure CSI Drivers in non-default namespace
 
-If you've deployed the Rook operator in a namespace other than "rook-ceph",
+If you've deployed the Rook operator in a namespace other than `rook-ceph`,
 change the prefix in the provisioner to match the namespace you used. For
-example, if the Rook operator is running in the namespace "my-namespace" the
-provisioner value should be "my-namespace.rbd.csi.ceph.com". The same provisioner
-name needs to be set in both the storageclass and snapshotclass.
+example, if the Rook operator is running in the namespace `my-namespace` the
+provisioner value should be `my-namespace.rbd.csi.ceph.com`. The same provisioner
+name must be set in both the storageclass and snapshotclass.
 
 ## Liveness Sidecar
 
-All CSI pods are deployed with a sidecar container that provides a prometheus metric for tracking if the CSI plugin is alive and running.
-These metrics are meant to be collected by prometheus but can be accesses through a GET request to a specific node ip.
-for example `curl -X get http://[pod ip]:[liveness-port][liveness-path]  2>/dev/null | grep csi`
-the expected output should be
+All CSI pods are deployed with a sidecar container that provides a Prometheus
+metric for tracking whether the CSI plugin is alive and running.
+These metrics are meant to be scraped (collected) by Prometheus but can also be
+accessed through a GET request to a specific node,.
+for example `curl -X get http://[pod ip]:[liveness-port][liveness-path]  2>/dev/null | grep csi`.
```

Comment:
Let's put this command in a separate code block instead of embedded in the paragraph (put triple ticks around this):
```suggestion
curl -X get http://[pod ip]:[liveness-port][liveness-path]  2>/dev/null | grep csi
```
```